### ISSUE TITLE
Got problem when customizing default with TypeDef

### ISSUE THREAD

**Issue by CKRainbow** [dbeckwith/rust-typescript-type-def#1]
According to the serde doc, the code below for customizing default for field is valid: 
```
#[derive(Deserialize, Debug)]
struct Request {
    #[serde(default = "default_resource")]
    resource: String,
}

fn default_resource() -> String {
    "/".to_string()
}
```
But when I add TypeDef to derive, there will be an error says
```
Unknown literal value `default_resource`
```

What's going wrong?

**Comment by dbeckwith** [comment]
Currently the derive macro only supports `#[serde(default)]`, not `#[serde(default = "default_resource")]`. It shouldn't be hard for me to add though. I also need to better document which serde attributes are/aren't supported.

**Comment by dbeckwith** [comment]
Fixed in v0.2.3
